utils/create-news-verification-hashes/index.js

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
wouldn't this file fit butter under `/scripts/`?

Reply:
@daniloprates I agree but since we have `v2-api-import` script in utils `v2-integration branch` I decided to put inside utils. Should I change that, what you think?